### PR TITLE
adding a team page

### PR DESCRIPTION
**Pull request by choldgraf**
This adds an "Our team" section to the 2i2c About page. This is now _in addition to_ the founders page. It also includes a "Volunteers" section to make it clear where people are being paid as 2i2c staff, and where they're volunteering time (@yuvipanda are you OK with this?).

Feel free to provide comments and thoughts if you think it should be different. I am mostly making this PR to have something we can look at and agree upon. It does not have to be set in stone.

@GeorgianaElena and @consideRatio are you OK with how this looks?

[Here's a preview of the section](https://deploy-preview-29--cocky-booth-e7ed17.netlify.app/about/#our-team)

closes https://github.com/2i2c-org/meta/issues/79

**ps anybody that wants a bio:** should sent me a short 1-2 paragraph bio and I'll add it

### CODE REVIEW

**Comment by consideRatio** [comment]
@choldgraf I'm happy about how this looks! Thank you for your work!

**Comment by GeorgianaElena** [comment]
TY :heart_eyes:

**Review by yuvipanda — APPROVED** [review]
<3 ty

**Comment by rabernat** [comment]
Thanks for this Chris. I fixed the spelling of my name. 😉

**Review by rabernat — APPROVED** [review]
Is it weird that I'm the only advisory board member with social media links?

File: content/authors/georgiana-dolocan/_index.md [hunk 1]
```diff
@@ -49,3 +49,5 @@ user_groups:
 - Open Engineering Team
 - Staff
 ---
+
+Software Engineer irreversibly in love with open source. A JupyterHub team member, focusing on infrastructure and community growth. Previously JupyterHub Contributor in Residence and Outreachy intern through an internship that supports diversity in open source and free software.
```
Comment (consideRatio) [inline content/authors/georgiana-dolocan/_index.md:53]: _"irreversibly in love with open source"_ made me smile wide :heart_eyes:

Comment (GeorgianaElena) [reply content/authors/georgiana-dolocan/_index.md:53]: Awww :heart:  :heart_eyes: :heart: I'm so happy :heart: This was the first sentence that came to my mind when trying to describe the me-work relationship.

**Pull request merged**